linode/manager
# refactor [M3-6262]: MUI v5 Migration - `SRC > Features > Domains`

## Description 📝
Migrates SRC > Features > Domains from JSS to tss-react (Emotion) / styled components

## Major Changes 🔄
- Removed @mui/styles imports and converted most styling to styled components
- Removed React.FC
- removed default exports for functional components in favor of inline exports (had to use an intermediate module for any components using React.lazy)
- renamed component props to be more specific

## How to test 🧪
- go to the /domains section locally and ensure all functionality works and that things look the same
  - note: there seems to be a slight difference in width for the first table cell from the landing page (like below screenshot) on local compared to the cloud manager site, but the discrepancy likely isn't related to this specific PR, as this difference also appears on other tables (volumes, etc)
![image](https://github.com/linode/manager/assets/139280159/a2da560b-f5cb-4134-9d70-a4a4472aa10b)
